### ISSUE TITLE
Fix the Modal component so it renders in Fractal on $ yarn start

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/design-system#218]
**Describe the bug**
After pulling the `master` branch of our Formation design system and running pre-requisites outlined below, the Modal component would never load and gave the following console errors:

`components.bundle.js:54285 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

`Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

I tried several different ways to import the component into the Nunjucks template, and double checked that the default export was correct. I also re-ran the unit tests to ensure proper loading into the test file.

**To Reproduce**
1. Pulled `master` branch of [design-system repo](https://github.com/department-of-veterans-affairs/design-system)
2. Entered $ yarn install` on terminal
3. Entered `$ yarn start` on terminal
4. Opened **localhost:3000** in browser
5. Clicked on Modal--default link ( http://localhost:3000/components/detail/modal--default )
6. Opened Chrome dev tools and confirmed error messages on Console

**Expected behavior**
I expected the modal components to load into the page properly.

**Screenshots**
<img width="764" alt="screen shot 2019-01-09 at 10 34 00 am" src="https://user-images.githubusercontent.com/934879/50913499-1ebd8780-13fa-11e9-83a3-a2aa8cde19d0.png">

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser: Chrome latest

**Additional context**
I'm running an audit of the Recommendations MVP and have an issue that could be improved by editing this component. I have to wait for the issue to be fixed and promoted to the test environment without the Modal component working properly.

**Comment by jaredcunha-usds** [comment]
Out of curiousity, is this related to other things not working either? 

[Buttons](https://department-of-veterans-affairs.github.io/design-system/components/detail/buttons.html), for instance.
<img width="626" alt="screen shot 2019-01-09 at 11 45 09 am" src="https://user-images.githubusercontent.com/25435289/50914268-26822980-1404-11e9-98bf-5f5195b52975.png">

Maybe not though? I don't remember the modals ever working but the buttons did at some point.

**Comment by 1Copenut** [comment]
It's possible, but I didn't look much past Modals. I'll see if the same console error is showing on the Buttons view.

**Comment by 1Copenut** [comment]
Looks like the buttons are more of a missing stylesheet. I'm seeing all the buttons in the expected styles and presentation on localhost.
